### PR TITLE
Add new case install_uninstall/downgrade_upgrade driver for virtio mouse/tablet device

### PR DESCRIPTION
**Pull request by peixiu**
Add 4 new cases:
win_virtio_driver_update_test..device_mouse..uninstall_install/downgrade_upgrade
win_virtio_driver_update_test..device_tablet..uninstall_install/downgrade_upgrade

id: 1786405, 1786406, 1786407, 1786408
Signed-off-by: Peixiu Hou <phou@redhat.com>

### CODE REVIEW

**Comment by peixiu** [comment]
(3/6) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_mouse.before_bg_test.uninstall_install.q35: PASS (185.99 s)
 (4/6) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_mouse.before_bg_test.downgrade_upgrade.q35: PASS (217.97 s)
 (5/6) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_tablet.before_bg_test.uninstall_install.q35: PASS (189.02 s)
 (6/6) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_tablet.before_bg_test.downgrade_upgrade.q35: PASS (219.24 s)

**Comment by peixiu** [comment]
based on patch https://github.com/autotest/tp-qemu/pull/1902

**Comment by peixiu** [comment]
@luckyh Please help to review this patch when you free, thanks a lot~

**Comment by peixiu** [comment]
@luckyh please help to review it when you free, it based on patch https://github.com/autotest/tp-qemu/pull/1902, thanks~

File: qemu/tests/cfg/win_virtio_driver_update_test.cfg [hunk 1]
```diff
@@ -189,8 +189,23 @@
             device_hwid = '"PCI\VEN_1AF4&DEV_1052"'
             inputs = input1
             input_dev_bus_type_input1 = virtio
+            del usb_devices
+            move_rate = 80
+            move_duration = 1
+            btns = "left right middle side extra"
+            scrolls = "wheel-up wheel-down"
```
Comment (luckyh) [inline qemu/tests/cfg/win_virtio_driver_update_test.cfg]: see https://github.com/autotest/tp-qemu/pull/2042#discussion_r382550601

Comment (peixiu) [reply qemu/tests/cfg/win_virtio_driver_update_test.cfg]: updated the code, thanks~

**Comment by peixiu** [comment]
Updated the code, and tested with latest code on win2012-r2 guest, all passed:

 (13/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_keyboard.before_bg_test.uninstall_install.q35: PASS (367.01 s)
 (14/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_keyboard.before_bg_test.downgrade_upgrade.q35: PASS (163.56 s)
 (15/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_mouse.before_bg_test.uninstall_install.q35: PASS (120.90 s)
 (16/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_mouse.before_bg_test.downgrade_upgrade.q35: PASS (144.91 s)
 (17/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_tablet.before_bg_test.uninstall_install.q35: PASS (121.02 s)
 (18/18) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_tablet.before_bg_test.downgrade_upgrade.q35: PASS (144.15 s)

**Comment by peixiu** [comment]
@luckyh Please help to review it again, thanks a lot~

**Comment by lijinlijin** [comment]
For case logic, LGTM
Acked-by: Li Jin lijin@redhat.com

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**